frontend/components/DataSet/_styles.scss

```diff
@@ -1,7 +1,18 @@
 .data-set {
   font-size: $x-small;
+  min-width: max-content;
```

Comment:
Do these changes still work for the places this component is already in use? Software details summaries I think?

Reply:
Yeah, this was added to prevent some long lines were being wrapped.